### PR TITLE
fixed markup, fixed routes, fixed route tests for showQuiz Angular template

### PR DESCRIPTION
**Pull request by brouhaha88**
[Link to JIRA ticket](https://ssu-jira.softserveinc.com/browse/LR-208)

### Author.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [x] @YUzhva 
- [x] @OKrain 
- [x] @vovka

### Issue summary.

markup was improper

### Fix summary. 

fixed markup, fixed routes and tests

### Testing approach.

unit testing

### CODE REVIEW

File: app/assets/javascripts/angularjs_app/components/quizzes/views/showQuiz.html [hunk 1]
```diff
@@ -1,21 +1,17 @@
-<h1>
-  <strong>Quiz name:</strong>
-</h1>
 <ul class="list-group">
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/views/showQuiz.html]: Why header is inside list?

File: app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js [hunk 2]
```diff
@@ -80,5 +80,6 @@ quizzesControllers.controller('startQuizCtrl', ['$scope', '$routeParams', 'Quiz'
 quizzesControllers.controller('showQuizCtrl', ['$scope', 'Quiz', '$routeParams',
   function($scope, Quiz, $routeParams) {
     $scope.quiz = Quiz.show({quizId: $routeParams.quizId});
+    $scope.time = 'seconds';
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: I believe you may put this string directly in view, no need to declare model.

Comment (brouhaha88) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: yep)

**Comment by vovka** [comment]
🆗

**Pull request merged**